metastats-infrastructure/src/main/groovy/life/qbic/metastats/filter/JsonValidator.groovy

```diff
@@ -50,7 +51,8 @@ class JsonValidator implements SchemaValidator {
         report = jsonSchema.validate(node, true)
 
         if (!report.isSuccess()) {
-            LOG.info report
+            LOG.warning "Validation failure! Check the log file to see a detailed validation error."
+            LOG.info report as String
         }
```

Comment:
Replace with try catch block, log or propagate error.

Reply:
I do not desire to have an exception. I want to inform the user about problems in their project. If I abort the final document will not be generated.